src/main/java/org/sonarlint/intellij/config/global/wizard/ServerStep.java

```diff
@@ -94,6 +107,17 @@ protected void textChanged(DocumentEvent e) {
 
     nameField.setToolTipText("Name of this configuration (mandatory field)");
 
+    var version = getIntelliJVersion();
+
+    if(isVersionHigher(version, "2024.2")) {
+      proxyWarningText.setEnabled(true);
```

Comment:
Here you put `proxyWarningText.setEnabled(true);` and below `proxyWarningText.setVisible(false);`

Shouldn't those two statements somewhat match? eg, both should be setEnable or setVisible?